resources/js/gallery.js

```diff
@@ -57,6 +57,13 @@ humhub.module('gallery', function (module, require, $) {
         });
     };
 
+    $('#gallery-media-container .panel-body').one('click', function() {
```

Comment:
Note that the gallery.js is only loaded once, so this event handler won't be attached when leaving and reentering the gallery view. Such handlers should be attached within the `init` function.

Reply:
> Note that the gallery.js is only loaded once, so this event handler won't be attached when leaving and reentering the gallery view. Such handlers should be attached within the `init` function.

Oh, thank you, @buddh4!

Yeah, for sure, you are right!
I will fix it (by moving this part to `init` function) today a bit later.